src/pydvl/influence/base_influence_model.py

```diff
@@ -0,0 +1,173 @@
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import Collection, Generic, Iterable, Optional, TypeVar
+
+
+class InfluenceType(str, Enum):
+    r"""
+    Enum representation for the types of influence.
+
+    Attributes:
+        Up: Up-weighting a training point, see section 2.1 of
+            (Koh and Liang, 2017)<sup><a href="#koh_liang_2017">1</a></sup>
+        Perturbation: Perturb a training point, see section 2.2 of
+            (Koh and Liang, 2017)<sup><a href="#koh_liang_2017">1</a></sup>
```

Comment:
Don't we have this in our documentation? It's always better to link to it. Going to a paper for details is very cumbersome